### PR TITLE
feat(imgur.txt): refresh the list by 211126

### PR DESCRIPTION
**Pull request by tai271828**
Issue: #1

The execution result shows

```
Read lastResult file........                                                                                                                                                                                                                 
Read imgur.txt......
There are valid 0 files, and need to update 3120 
Saved - output/SP8RBV2.jpg
Saved - output/B1TdSL0.jpg
Saved - output/YMX0Mwg.jpg
Saved - output/2FDkXVM.jpg
Saved - output/c2DXT5o.png

<skipped>

Saved - output/A3Xrigc.jpg
Saved - output/RNrBYDF.jpg
Saved - output/3F4vWZd.jpg
Saved - output/cdJsxVf.jpg
Saved - output/YYBVySY.jpg
Saved - output/TQDzEhZ.jpg
Saved - output/6GTXXRx.jpg
Saved - output/cLq8YdD.jpg
Saved - output/n0nTLNy.jpg
✨ Success



┌─[^_^]─[~/work-my-projects/imgur-keeper]
  tai271828@syakaro: 15 file 660Kb
└──╼ ls output/ | wc -l
3119
```

The 3120 vs. 3119 seems to be caused by the duplicate entry Grqb5xy.jpg

### CODE REVIEW

File: imgur.txt [hunk 1]
```diff
@@ -492,8 +502,6 @@ https://i.imgur.com/U5cG24w.jpg
 https://i.imgur.com/AWSUMUy.jpg
 https://i.imgur.com/dAhatL3.jpg
 https://i.imgur.com/IfhwM4G.jpg
-https://i.imgur.com/sdrisC2.jpg
```
Comment (tai271828) [inline imgur.txt:495]: Checked the two missing items and the removal is expected.

**Comment by tai271828** [comment]
@yoyo930021 It's my first time to scan the imgur urls. Would you mind reviewing what I have done (the output message and the flow) if you have a moment?
@deeper747 I was aware that some images were removed. I checked some of them randomly but not all of them. Would you mind reviewing the missing images to see if they fit your expectation?

File: imgur.txt [hunk 2]
```diff
@@ -1932,7 +3115,6 @@ https://i.imgur.com/Y5JbxBv.png
 https://i.imgur.com/KTNCZLe.jpg
 https://i.imgur.com/0OrBtEB.png
 https://i.imgur.com/bNBXBj0.png
-https://i.imgur.com/8RQ3bXc.jpg
```
Comment (deeper747) [inline imgur.txt:1935]: This was deleted by me these month since it was a testing file.

File: imgur.txt [hunk 3]
```diff
@@ -790,8 +1975,6 @@ https://i.imgur.com/TrAsbQA.jpg
 https://i.imgur.com/EUym73A.jpg
 https://i.imgur.com/jHfap6m.jpg
 https://i.imgur.com/ctfs8Qj.jpg
-https://i.imgur.com/zrlVQlG.jpg
-https://i.imgur.com/r1CKXK1.jpg
```
Comment (deeper747) [inline imgur.txt:794]: deleted /cause repetitive upload

**Comment by deeper747** [comment]
> @deeper747 I was aware that some images were removed. I checked some of them randomly but not all of them. Would you mind reviewing the missing images to see if they fit your expectation?

Randomly checked the highlighted items. All are deleted in purpose. 
No problem so far in my opinion!

**Review by deeper747 — APPROVED** [review]

**Pull request merged**